src/vmm/src/devices/virtio/vhost_user_block/device.rs

```diff
@@ -291,6 +296,7 @@ impl<T: VhostUserHandleBackend + Send + 'static> VirtioDevice for VhostUserBlock
         let config_len = self.config_space.len() as u64;
         if offset >= config_len {
             error!("Failed to read config space");
+            self.metrics.cfg_fails.inc();
```

Comment:
there are quite a few things that may go wrong in vhost-user. Why did you choose this specific one?

Reply:
I added this by referring to [this similar](https://github.com/search?q=repo%3Asudanl0%2Ffirecracker%20%22metrics.cfg_fails.inc()%3B%22&type=code) metric for block and net device.

Reply:
no, not quite, I guess those two time metrics are the most interesting at the moment.